src/nvim/memline.c

```diff
@@ -3100,23 +3013,20 @@ char_u *makeswapname(char_u *fname, char_u *ffname, buf_T *buf, char_u *dir_name
 /// @param dname  don't use "dirname", it is a global for Alpha
 char *get_file_in_dir(char *fname, char *dname)
 {
-  char *t;
-  char *tail;
   char *retval;
-  int save_char;
 
-  tail = path_tail(fname);
+  char *tail = path_tail((char *)fname);
```

Comment:
Forgot about this one
```suggestion
  char *tail = path_tail(fname);
```